src/assets/stylesheets/index.scss

```diff
@@ -19,20 +19,20 @@
   --border: rgba(0, 0, 0, 0.1);
 
   // The variable var(--bs-primary) is utilized in packages like `scrivito-neoletter-form-widgets`.
-  --bs-primary: #274486;
+  --bs-primary: #072b46; // #274486 before
```

Comment:
Please remove the `// #274486 before` comment. It's no longer needed.

Reply:
Done with latest commit